docs/earthly-command/earthly-command.md

```diff
@@ -646,6 +646,148 @@ List secrets the current account has access to.
 
 Removes a secret from the secrets store.
 
+## earthly registry
+
+Contains sub-commands for managing registry access in cloud-based secrets.
+
+#### earthly registry setup
```

Comment:
Did you mean to use 3 levels here?

```suggestion
### earthly registry setup
```

Reply:
good catch, I started adding two levels per subheading :facepalm: